docs/user_guide/default_science_configurations.md

```diff
@@ -7,35 +7,31 @@ The science configurations are given as patches to apply to a default namelist f
 Currently, the default science configurations are defined internally by the following data structure:
 ```python
 DEFAULT_SCIENCE_CONFIGURATIONS = {
-    "sci0": {"cable": {"cable_user": {"GS_SWITCH": "medlyn"}}},
-    "sci1": {"cable": {"cable_user": {"GS_SWITCH": "leuning"}}},
-    "sci2": {"cable": {"cable_user": {"FWSOIL_SWITCH": "Haverd2013"}}},
-    "sci3": {"cable": {"cable_user": {"FWSOIL_SWITCH": "standard"}}},
-    "sci4": {
+    "sci0": {
```

Comment:
@ccarouge did you prefer to keep the "sciX" labels in the documentation? I was planning on removing these and copying the data structure from internal as verbatim

Reply:
I would put a label in the documentation so the relation to the results is made clear to the users. Maybe go with "S0", etc. instead since that is what is used for the files?

Reply:
Would something like the following work? I prefer to keep the list syntax as this is consistent with how they are specified in the config file:
```
DEFAULT_SCIENCE_CONFIGURATIONS = [
    {  # S0 configuration
        "cable": {
            "cable_user": {
                "GS_SWITCH": "medlyn",
                "FWSOIL_SWITCH": "Haverd2013",
            }
        }
    },
    {  # S1 configuration
        "cable": {
            "cable_user": {
                "GS_SWITCH": "leuning",
                "FWSOIL_SWITCH": "Haverd2013",
            }
        }
    },
    {  # S2 configuration
        "cable": {
            "cable_user": {
                "GS_SWITCH": "medlyn",
                "FWSOIL_SWITCH": "standard",
            }
        }
    },
    {  # S3 configuration
        "cable": {
            "cable_user": {
                "GS_SWITCH": "leuning",
                "FWSOIL_SWITCH": "standard",
            }
        }
    },
]
```